src/java.desktop/share/classes/javax/swing/JScrollBar.java

```diff
@@ -773,6 +798,9 @@ public Dimension getMinimumSize() {
      * rigid along the other axis.
      */
     public Dimension getMaximumSize() {
```

Comment:
Since we are here, you should fix this to add the @return. This does not need to be part of the CSR

Reply:
Added @param and @return